File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.js
function solution(a, b) {
    var answer = 0;
    let temp1 = Number(String(a) + String(b))
    let temp2 = 2 * a * b
    
    if(temp1 >= temp2){
        answer = temp1
    }
    else{
        answer =temp2
    }
    return answer;
}

File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.java
class Solution {
    public int solution(int a, int b) {
        int temp1 = Integer.parseInt(Integer.toString(a) + Integer.toString(b));
        int temp2 = 2 * a * b;
        
        if(temp1 < temp2){
            return temp2;
        }
        else{
            return temp1;
        }
    }
}